### ISSUE TITLE
[Feature Request] Add loging

### ISSUE THREAD

**Issue by BennyAlex** [muratcorlu/connect-api-mocker#53]
Hey @muratcorlu 
It would be nice if you could turn on logging, displaying informations like the request url and what happend with it(json file loaded/ forwarded because no mock was found etc)

**Comment by muratcorlu** [comment]
Do you think current logging functionality is not enough or you didn't notice it? https://github.com/muratcorlu/connect-api-mocker#logging

**Comment by BennyAlex** [comment]
@muratcorlu 
didnt saw it. But it only logs if a mock file was found, for forwarded requests it would be also nice to see an information :)

**Comment by muratcorlu** [comment]
Actually connect-api-mocker doesn't forward requests, just pass to the next middleware. So if you have another middleware that handles same url to go to real api it will be forwarded. But a convenient log message could be put to that line https://github.com/muratcorlu/connect-api-mocker/blob/master/index.js#L158 . Maybe something like `Couldn't find a mock for this url. Request passing to next middleware.` Is this what you mean?

**Comment by BennyAlex** [comment]
@muratcorlu 
yes, display` Couldn't find a mock for this url.` and `Request passing to next middleware.` if also `nextOnNotFound `is set to true.

Thanks.
